### ISSUE TITLE
Error if parsing IP address with scope id in the end.

### ISSUE THREAD

**Issue by wolgasster** [jsakamoto/ipaddressrange#32]
Cannot parse IP addresses with '%' sign, e.g. "fe80::8430:a7f7:5f9c:7c2c%6"
(System.Net.IPAddress parses it without exception).

**Comment by jsakamoto** [comment]
Oh...

Thank you for your report 👍 

I'm going to fix it until next week.

**Comment by jsakamoto** [comment]
I fixed it at commit 8b8058dc7271412628a6dd2c54e4032301aeb801.

Notice: `Begin` and `End` properties in `IPAddressRange` object always don't containing scoped id if those are parsed from an IP address string with scope id.   
Because "range" doesn't rely on individual the network card. 

I released the fixed version as a Beta release.

https://www.nuget.org/packages/IPAddressRange/2.1.1-beta

I'll release as an official version in a few days if there are no more problems.
